arm/Microsoft.ServiceBus/namespaces/authorizationRules/deploy.bicep

```diff
@@ -32,7 +37,7 @@ resource authorizationRule 'Microsoft.ServiceBus/namespaces/AuthorizationRules@2
 @description('The name of the authorization rule.')
 output authorizationRuleName string = authorizationRule.name
 
-@description('The Resource Id of the authorization rule.')
+@description('The Resource ID of the authorization rule.')
```

Comment:
As per other output and param descriptions convert to lowercase except for ID

Reply:
Fixed